src/pybind/mgr/cephadm/module.py

```diff
@@ -1980,6 +2023,18 @@ def _create_osd(self, host, cmd):
         self.cache.invalidate_host_devices(host)
         return "Created osd(s) on host '{}'".format(host)
 
+    def _calc_daemon_deps(self, daemon_type, daemon_id):
+        need = {
+            'prometheus': ['mgr', 'alertmanager', 'node-exporter'],
```

Comment:
shouldn't this just be `node-exporter` ?

it would seem like `mgr` and `alertmanager` depend on `prometheus` ..

Reply:
the prom config references mgrs (to scrape ceph), node-exporters (for hosts), and alertmanager (not sure why).  I'm just going from the _generate_*_config() functions and looking at what they put in the conf.